crates/optimism/node/src/txpool.rs

```diff
@@ -315,29 +315,14 @@ where
         }
     }
 
-    /// Validates a single transaction.
-    ///
-    /// See also [`TransactionValidator::validate_transaction`]
-    ///
-    /// This behaves the same as [`EthTransactionValidator::validate_one`], but in addition, ensures
-    /// that the account has enough balance to cover the L1 gas cost.
-    pub fn validate_one(
+    /// Validates the gas fees of a transaction
+    pub fn validate_gas_fees(
         &self,
-        origin: TransactionOrigin,
-        transaction: Tx,
+        outcome: TransactionValidationOutcome<Tx>,
     ) -> TransactionValidationOutcome<Tx> {
-        if transaction.is_eip4844() {
-            return TransactionValidationOutcome::Invalid(
-                transaction,
-                InvalidTransactionError::TxTypeNotSupported.into(),
-            )
-        }
-
-        let outcome = self.inner.validate_one(origin, transaction);
-
         if !self.requires_l1_data_gas_fee() {
             // no need to check L1 gas fee
-            return outcome
+            return outcome;
```

Comment:
```suggestion
            return outcome
```
pls revert all these unrelated formatting changes. take care to use `cargo +nightly fmt` and not `cargo fmt`.